src/Symfony/Component/Form/Extension/Core/DataTransformer/StringToFloatTransformer.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+class StringToFloatTransformer implements DataTransformerInterface
+{
+    /**
+     * @var int|null
+     */
+    private $scale;
+
+    /**
+     * @param int|null $scale
+     */
```

Comment:
can be removed, no value according to typehinted arg, same for the property above.